process.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+    "fmt"
+    "os"
+    "path/filepath"
+    "io/ioutil"
+    "strings"
+    "strconv"
+    "sync"
+)
+
+func main() {
+    args := os.Args
+    sourceDir, err := filepath.Abs(args[1])
+    if err != nil {
+        fmt.Printf("error: %v\n", err)
+        os.Exit(0);
+    }
+    destDir, err := filepath.Abs(args[2])
+    if err != nil {
+        fmt.Printf("error: %v\n", err)
+        os.Exit(0);
+    }
+
+    if _, err := os.Stat(destDir); os.IsNotExist(err) {
+        fmt.Println(destDir)
+        os.Mkdir(destDir, 0777)
+    }
+
+    files, err := ioutil.ReadDir(sourceDir)
+    if err != nil {
+        fmt.Printf("error: %v\n", err)
+        os.Exit(0);
+    }
+
+    var wg sync.WaitGroup
+
+    countFileLines := func(fileName, sourceDir, destDir string, wg *sync.WaitGroup) {
```

Comment:
It's not necessary to pass the WaitGroup as an argument to a function because it'll be visible from the context. You can try to delete the `wg` argument and it will work as well